zanata-war/src/main/java/org/zanata/action/CopyrightAction.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015, Red Hat, Inc. and individual contributors as indicated by the
+ * @author tags. See the copyright.txt file in the distribution for a full
+ * listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This software is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA, or see the FSF
+ * site: http://www.fsf.org.
+ */
+package org.zanata.action;
+
+import org.jboss.seam.ScopeType;
+import org.jboss.seam.annotations.In;
+import org.jboss.seam.annotations.Name;
+import org.jboss.seam.annotations.Scope;
+import org.zanata.i18n.Messages;
+
+import java.io.Serializable;
+import java.util.Calendar;
+
+/**
+ * @author Damian Jansen
+ * <a href="mailto:djansen@redhat.com">djansen@redhat.com</a>
+ */
+@Name("copyrightAction")
+@Scope(ScopeType.PAGE)
+public class CopyrightAction implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @In
+    private Messages msgs;
+
+    public String copyrightNotice() {
+        return msgs.format("jsf.CopyrightNotice",
+                String.valueOf(Calendar.getInstance().get(Calendar.YEAR)));
+    }
+}
```

Comment:
Don't think we need an action bean just for getting year. You can implement the method in applicationConfiguration bean.